movement/roi/base.py

```diff
@@ -0,0 +1,159 @@
+"""Class for representing 2-dimensional regions of interest (RoIs)."""
```

Comment:
This should say "1- or 2- dimensional regions of interest" or just "regions of interest (RoIs)".

Reply:
```suggestion
"""Class for representing 1- or 2-dimensional regions of interest (RoIs)."""
```